device-types/TrippLite/SMART3000RMXL2U.yaml

```diff
@@ -0,0 +1,72 @@
+manufacturer: Tripplite
+model: SMART3000RMXL2U
+slug: smart3000rmxl2u
+part_number: ''
+u_height: 2.0
+is_full_depth: true
+subdevice_role: ''
+airflow: front-to-rear
+comments: ''
+power-ports:
+- name: Plug1
```

Comment:
This name could be better chosen, is there an official name for this (perhaps input instead of Plug1)

Reply:
Compared with other tripplite submissions and changed to primary to keep consistency.